### ISSUE TITLE
Problém s originInfo u periodik s více vydavateli/místy vydání - AACR i RDA záznamy

### ISSUE THREAD

**Issue by luckajirku** [proarc/proarc#739]
Sem to dávám spíš pro info, kdyby na to někdo narazil, problém je už v šabloně, na standardech jsem k tomu založila issues:
u AARC záznamů: https://github.com/NLCR/Standard_NDK/issues/52
u RDA záznamů: https://github.com/NLCR/Standard_NDK/issues/53

**Comment by luckajirku** [comment]
Tak jelikož si máme každý upravovat svoji šablonu sám, je třeba to řešit tady. Kopíruju sem text z těch dvou issues:

**OriginInfo u periodik s více vydavateli/místy vydání - AACR záznam**
V AACR záznamu periodika máme několik výskytů pole 260, jak se postupem let měnil vydavatel a místo vydání.
Z MARCu do MODSu se převedou všechny údaje do jednoho originInfo, kde jsou nejprve všechny výskyty place, pak všechny publisher, pak dateIssued atd., místo aby se pro každý výskyt 260 vytvořil samostatný originInfo, ve kterém by byly údaje jen z toho jednoho výskytu pole 260.

**OriginInfo u periodik s více vydavateli/místy vydání - RDA záznam**
V RDA záznamu periodika máme několik výskytů pole 264, první má roky v podpoli c a u těch dalších jsou roky v podpoli 3.
V MODSu se ten první výskyt převede správně, roky se převedou do dateIssued, ale s převodem roků z podpole 3 šablona nepočítá.
Protože druhý indikátor i v těch dalších výskytech je 1, všude se vytvoří eventType:publication, a tím pádem to vyžaduje i vyplnění dateIssued.
Ani v poslední šabloně LoC (3.6) převod roků z podpole 3 není, je možné prověřit proč?

**Comment by luckajirku** [comment]
A tohle je potřeba opravit @SykoraLukas

**Comment by luckajirku** [comment]
stanovisko NK je od r. 2019 tady v tom issue: https://github.com/NLCR/Standard_NDK/issues/53, asi to sem překopíruju:
"informovala som sa v LoC ohľadne zmieneného problematického poľa. Ich mapping MARC21toMODS to nerieši a (asi?) ani nebude, ale poradili mi riešenie, a to dávať túto informáciu do dateOther a použiť atribút type s hodnotou napr. earliest, latest atď... na indikovanie, o aké časové rozpätie sa jedná. My si podľa toho šablónu upravovať nebudeme, ale vám toto riešenie možno pomôže."

**Comment by SykoraLukas** [comment]
Šablonu muzu upravit ale chtelo by to, aby se to domluvilo společně s issue #1266 a #1267.

**Comment by SykoraLukas** [comment]
Mohl by prosím někdo sepsat (@nezbedova, @tumovam, @luckajirku), jak se to má chovat u periodik/monografii v zavislosti na jednotlivých pravidlech?

Zda se to má dávat do jednoho originInfo nebo se má založit co záznam to nove originInfo? Děkuji

**Comment by SykoraLukas** [comment]
Chápu, ale dle vyjádření @luckajirku v https://github.com/proarc/proarc/issues/1266 to je právě přesně naopak.

**Comment by tumovam** [comment]
Nebudu to motat sem, protože tady jsou periodika. Více v 1266...

**Comment by SykoraLukas** [comment]
No ale ono to spolu souvisí, protože šablona je jen jedna jak pro periodika tak pro monografie a je potřeba se domluvit, jak se to bude dělat. Nelze udělat 2 šablony.

**Comment by tumovam** [comment]
@luckajirku v 1266 odpovídá, že "... tam je totiž naopak násobné origininfo žádoucí". Takže já to chápu tak, že nám oběma jde o totéž. Že všichni chceme více originInfo :)) LJ, můžu vás poprosit o vyjádření, rozumíme si?

**Comment by luckajirku** [comment]
ten problém tam je dvojí a JE důležité, jestli to je AACR, nebo RDA (viz https://github.com/proarc/proarc/issues/739#issuecomment-438159589)

**Comment by luckajirku** [comment]
takže je potřeba vyřešit:
1.  vznik samostatného originInfo pro každý výskyt 260/264
2. u RDA záznamů ještě převod roků z 264, podpole 3. Podle toho, co psala NK, asi teda roky z 264c dál převádět do dateIssued, ale roky z 264, podp. 3 do dateOther s atributy earliest, latest... já se na to ještě podívám v záznamech, jak moc jde tenhle atribut zautomatizovat, ale nechtějte to po mě v pátek v poledne:-)

**Comment by SykoraLukas** [comment]
Ten první bod je v pohodě, takže pokud s tím souhlasíte, tak to doplním. 

Druhý bod doplním až mi řeknete, jak to přesně má být zautomatizované + do všech formulářů bych doplnil k poli dateOther možnost volby pro atribut, ať se to dá editovat i ve formuláři a ne jen v xml.

**Comment by ZdenkaSera** [comment]
Má se řešit společně s issue [#1266](https://github.com/proarc/proarc/issues/1266) ->> přidávám štítek "Rozšíření 2021".

**Comment by ZdenkaSera** [comment]
probrat mimo schůzku HK/KNAV

**Comment by luckajirku** [comment]
Tohle pořád není vyřešené. Viz tohle periodikum - sysno v katalogu SVK HK: 000162401. OriginInfo se udělá 2x, jedno je v pořádku, ve druhém je sesypaných dohromady 13 vydavatelů, x míst a frekvencí vydávání. A je to aacr, tam by problém s rozdělením do jednotlivých výskytů přeci být neměl. 

 
![origininfo](https://user-images.githubusercontent.com/6359479/200312905-f32a83df-28a0-4c8c-9104-1b0a0ffba81a.png)

**Comment by luckajirku** [comment]
a jak vypadají jednotlivé výskyty v bib  záznamu:
![jiskra_opac](https://user-images.githubusercontent.com/6359479/200326030-e8ad8f52-13b6-4a24-bf08-c7b46679984b.png)

**Comment by ZdenkaSera** [comment]
Nemám po ruce tak bohatý příklad, ale vyzkoušela jsem na KNAV datech sysno = 000614330, verze 3.9.1
Záznamu vypadá takto:
![image](https://user-images.githubusercontent.com/79305516/207057490-d31c67db-be39-4c18-98a2-40c529c2005e.png)

Podle mého by první 260 měla vést na 2 origininfa - to se nepovedlo. 
Druhá 260 vytvořila vlastní origininfo, to je ok, ale date isseud je 2x

![image](https://user-images.githubusercontent.com/79305516/207058762-6f6a48d4-cffe-4ff0-b7e4-2ea940d3e52d.png)

**Comment by ZdenkaSera** [comment]
@luckajirku Prosím, mýlím se nebo je to takto? Děkuji.
- každý výskyt 260 nebo 264 by měl vést na samostatné origininfo
- každý z těchto výskytů se pak bude případně dál dělit, pokud se bude opakovat kterékoli z podpolí a, b, c

**Comment by luckajirku** [comment]
no popravdě se v tom už dost ztrácím. Jednak by asi u těch výskytů z podpole 3 mělo být spíš dateOther než dateIssued. Nebo to v Krameriovi nebude vadit?
A jak s těmi případy, kdy je v jednom výskytu 260 třeba a b b a b, jestli takové pole fakt pak rozdělit do dvou samostatných origininfo, kde se zopakuje rok/roky pro obě ta místa, aby se k nim správě přiřadil vydavatel... Jinak to asi nepůjde, ale nevím.  Ta naše Jiskra východočeských horníků je hezký případ, na kterém je to všechno vidět.

**Comment by ZdenkaSera** [comment]
@luckajirku Děkuji za reakci. Jde mi hlavně o to, zda má vzniknout origininfo pro každé pole 260 nebo 264 - myslím, že to Lukáš rozporuje. 

- Ad "Dateissued": to v Krameriovi z hlediska zobrazení nevadí, s vyhledáváním ale vlastně nevím, musím zkusit. Na to ještě kouknu.
- Ad opakování podpolí a, b, c:  To jsme řešili v issue k monografiím - např. když to vydají 2 různé subjekty, oba z Brna a oba ve stejném roce, podpole a i c se zopakují pro každý ze subjektů. Ale určitě můžou nastat i nějaké komplikované situace, které současná úprava neošetří, skoro určitě ještě časem narazím. Příklad rozkopírování a, c je vidět třeba tady. 

![image](https://user-images.githubusercontent.com/79305516/207086390-5d7e0ac5-a874-4b27-8874-aeec92c297a6.png)

**Comment by SykoraLukas** [comment]
Nerozumíme si. Teď v aktuální verzi to je uděláno tak, že vznikne ke každému 260 nebo 264 vlastní originInfo. Pokud v Marc21 byl jen jeden 260 nebo 264 tak se pak řeší ještě rozdělení podle podpolí, pokud tam je nějaké vícekrát. Pokud je více 260 nebo 264 tak zůstává celý bok pospolu, neprobíhá rozdělení na základě podpolí.

**Comment by SykoraLukas** [comment]
Ano, nedává. Dříve to bylo tak, že se ze všech polí 260 udělalo jedno originInfo. Takže teď aspoň drříme to uspořádání podle toho, jak to je v katalogu.

**Comment by ZdenkaSera** [comment]
Předávám z issue https://github.com/proarc/proarc/issues/1564 sem

![image](https://github.com/proarc/proarc/assets/79305516/b2ca543e-6d18-4469-bc48-5ab3a1a7b0f1)

**Comment by SykoraLukas** [comment]
Implementováno.

**Comment by kerschfilip** [comment]
Issue zavírám a případné problémy budeme řešit při testování https://github.com/proarc/proarc/issues/1832
